app/Http/Controllers/DoctorController.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Filters\DoctorFilters;
+use App\Http\Requests\DoctorLoginRequest;
+use App\Http\Resources\TokenResource;
+use App\Services\DoctorService;
+use App\Traits\LoginTrait;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class DoctorController extends Controller
+{
+    use LoginTrait;
+
+    protected $doctorService;
+
+    public function __construct(DoctorService $doctorService)
+    {
+        $this->doctorService = $doctorService;
+    }
+
+    public function doctorLogin(DoctorLoginRequest $request , DoctorFilters $filters)
```

Comment:
- Name the function to `login`
- Do not inject the filters here and just call it in the repository
- Use Filters in the lists only